src/nile/core/init.py

```diff
@@ -14,6 +14,11 @@ def init():
     with open("accounts.json", "w") as file:
         file.write("{}")
 
+    logging.info("🔑  Creating .env file")
+
+    with open(".env", "w") as file:
+        file.write("# ALIAS = PRIVATE_KEY")
```

Comment:
what about adding it to the sample project directory?

Reply:
Good idea!